recipe/run_conda_forge_build_setup_osx

```diff
@@ -1,9 +1,5 @@
 #!/bin/bash
 
-# 1 core available on Travis CI OS X workers: https://docs.travis-ci.com/user/ci-environment/#Virtualization-environments
-# MAKEFLAGS is passed through conda build: https://github.com/conda/conda-build/pull/917
-export MAKEFLAGS="-j1 ${MAKEFLAGS}"
```

Comment:
Not that this is in a `conda-build` release yet, but just to get this ready for it. Can we please restore the comment above, newline below and change the `export` statement to `export CPU_COUNT=1`?